### ISSUE TITLE
Provide an AppImage for Linux

### ISSUE THREAD

**Issue by probonopd** [ONLYOFFICE/DesktopEditors#75]
**Do you want to request a *feature* or report a *bug*?**

Feature

**What is the current behavior?**

You need to install a deb or rpm which may or may not work with your Linux operating system.

**What is the intended behavior?**

Providing an [AppImage](http://appimage.org/) would have, among others, these advantages:
- Applications packaged as an AppImage can run on many distributions (including Ubuntu, Fedora, openSUSE, CentOS, elementaryOS, Linux Mint, and others)
- One app = one file = super simple for users: just download one AppImage file, [make it executable](http://discourse.appimage.org/t/how-to-make-an-appimage-executable/80), and run
- No unpacking or installation necessary
- No root needed
- No system libraries changed
- Works out of the box, no installation of runtimes needed
- Optional desktop integration with `appimaged`
- Optional binary delta updates, e.g., for continuous builds (only download the binary diff) using AppImageUpdate
- Can optionally GPG2-sign your AppImages (inside the file)
- Works on Live ISOs
- Can use the same AppImages when dual-booting multiple distributions
- Can be listed in the [AppImageHub](https://appimage.github.io/apps) central directory of available AppImages
- Can double as a self-extracting compressed archive with the `--appimage-extract` parameter

[Here is an overview](https://appimage.github.io/apps) of projects that are already distributing upstream-provided, official AppImages.

If you have questions, AppImage developers are on #AppImage on irc.freenode.net.

**Comment by vatterspun** [comment]
While I'm not specifically advocating AppImage over other current distribution methods, I will point out that LibreOffice has this functionality, whatever that's worth. 

https://www.libreoffice.org/download/appimage/

**Comment by probonopd** [comment]
Yes, if you'd like to try it out, here are many LibreOffice AppImages of various versions: http://libreoffice.soluzioniopen.com/

**Comment by ShockwaveNN** [comment]
We just released a first beta version of AppImage, you can try it out
https://github.com/ONLYOFFICE/appimage-desktopeditors

**Comment by probonopd** [comment]
Thank you very much. Passed my automated tests at https://travis-ci.org/AppImage/appimage.github.io/builds/385525065. Maybe you want to consider adding an [AppStream metadata file](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html) including a nice screenshot, so that it will show up nicely in https://appimage.github.io/apps/ and app centers that use this data.

Simple tool to make one:
http://output.jsbin.com/qoqukof

**Comment by ShockwaveNN** [comment]
@probonopd yeah thanks, we do this, I wanted to create PR to appimage.github.io after we moved out of beta, but since you've already done it - I add metadata soone

**Comment by ShockwaveNN** [comment]
@probonopd we have a look at AppStream metadata and think this is a great idea to add this not onlyo to AppImage, but also for our other distributives like deb and rpm and we will do it in next release

**Comment by probonopd** [comment]
Thank you. Please make absolutely sure that it passes `appstreamcli validate`.
